src/network/http_server.rs

```diff
@@ -1887,11 +1919,11 @@ impl HubHttpService for HubHttpServiceImpl {
                     }
                     return OnChainEvent {
                         r#type: match e.r#type {
-                            1 => OnChainEventType::EventTypeSigner,
-                            2 => OnChainEventType::EventTypeSignerMigrated,
-                            3 => OnChainEventType::EventTypeIdRegister,
-                            4 => OnChainEventType::EventTypeStorageRent,
-                            _ => OnChainEventType::EventTypeNone,
+                            1 => "SIGNER".to_string(),
```

Comment:
Is this to avoid the EVENT_TYPE prefix you'd get if you use the as_str... function?

Reply:
Yeah - is there a better way to be handling this?

Reply:
I think this is fine. in all the places where you're doing these manual conversions can you leave a note about the fact that we're doing this for compatibility. it's a bit odd so would be good to justify.

Reply:
Sure and yeah agreed. Another option would be to do the conversion then strip off the prefix but that seems more complex.  I'll add the comments at least.